manifests/proxy.pp

```diff
@@ -421,8 +421,7 @@
   # is set to for example "eth1" or "bond0.73".
   if ($listenip != undef) {
     if ($listenip =~ /^(eth|bond|lxc|eno|tap|tun).*/) {
-      $int_name  = "ipaddress_${listenip}"
-      $listen_ip = inline_template('<%= scope.lookupvar(int_name) %>')
+      $int_name  = getvar("::ipaddress_${listenip}")
```

Comment:
I mentioned that in https://github.com/voxpupuli/puppet-zabbix/issues/212. We should track that in a further PR.